charts/newrelic-infrastructure-beta/ci/test-values.yaml

```diff
@@ -0,0 +1,3 @@
+global:
```

Comment:
This shouldn't be "only global" and global should be more tested at `nri-bundle` level than here. But you should add as many values you want.

This only test that the manifests templates correctly. Not even if they has sense/are Kubernetes cluster. But stop lots of footguns.